rag_apps/simple_rag/.env.example

```
@@ -0,0 +1,67 @@
+﻿# simple_rag Environment Configuration
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The ﻿# simple_rag Environment Configuration key has incorrect delimiter

(IncorrectDelimiter)